lldb/source/Host/aix/Host.cpp

```diff
@@ -7,17 +7,139 @@
 //===----------------------------------------------------------------------===//
 
 #include "lldb/Host/Host.h"
+#include "lldb/Host/posix/Support.h"
+#include "lldb/Utility/LLDBLog.h"
+#include "lldb/Utility/Log.h"
+#include "lldb/Utility/ProcessInfo.h"
 #include "lldb/Utility/Status.h"
+#include "llvm/BinaryFormat/XCOFF.h"
+#include <sys/proc.h>
+#include <sys/procfs.h>
 
+using namespace llvm;
```

Comment:
I'd recommend not getting into the habit of including the whole llvm namespace. It may be fine if you have code that's heavily using some llvm objects, but in general I think it's more trouble than it's worth -- `llvm` is a big namespace and it includes names (e.g. `Module`) which conflict with lldb entities.

I don't think that's the case here, but if you find yourself using a particular function/class very often, then I it'd be better to just include that single class.